src/NexusMods.App.UI/Pages/ModLibrary/FileOrigins/FileOriginsPageViewModel.cs

```diff
@@ -199,7 +199,7 @@ private Task RegisterFileFromDisk(string path)
 
     public async Task OpenNexusModPage()
     {
-        var url = $"https://www.nexusmods.com/{_gameDomain.Value}";
+        var url = $"https://www.nexusmods.com/{_gameDomain.Value}?pk_source=nexusmodsapp";
```

Comment:
Depending on what gets merged first, either wait for #1557 or I'll include this in #1557.